common/src/main/java/com/wynntils/overlays/StatusEffectsOverlay.java

```diff
@@ -108,4 +136,74 @@ private void updateTextRenderSetting() {
     protected TextRenderSetting getTextRenderSetting() {
         return textRenderSetting;
     }
+
+    private Stream<RenderedStatusEffect> stackEffects(List<StatusEffect> effects) {
+        Map<String, RenderedStatusEffect> effectsToRender = new LinkedHashMap<>();
+
+        for (StatusEffect effect : effects) {
+            RenderedStatusEffect entry = effectsToRender.get(effect.asString().getString());
+            if (entry == null) {
+                entry = new RenderedStatusEffect(effect);
+                effectsToRender.put(effect.asString().getString(), entry);
+            }
+            entry.setCount(entry.getCount() + 1);
+        }
+        return effectsToRender.values().stream();
+    }
+
+    private static final class RenderedStatusEffect {
+        private int count = 0;
+        private final StatusEffect effect;
```

Comment:
```suggestion
        private final StatusEffect effect;

        private int count = 0;
```

finals and not finals are usually ordered and are separate